src/js/components/Legenda/index.js

```diff
@@ -0,0 +1,41 @@
+const legendas = [
+    {
+        label: 'FME',
+        legenda: 'Fundo Municipal de Educação',
+    },
+    {
+        label: 'HDEAYRE',
+        legenda: 'Frotinha do Ant Bezerra',
+    },
+    {
+        label: 'HDMSJB',
+        legenda: 'Frotinha da Parangaba',
+    },
+    {
+        label: 'HDNSCON',
+        legenda: 'Hosp. Nossa Sra Cj Ceará',
+    },
+    {
+        label: 'HEBO',
+        legenda: 'Frotinha da Messajana',
+    },
+    {
+        label: 'HDGM/ME',
+        legenda: 'Gonzaginha da Messajana',
+    },
+    {
+        label: 'GP',
+        legenda: 'Gabinete do Prefeito',
+    },
+    {
+        label: 'GVP',
+        legenda: 'Gabinete do Vice Prefeito',
+    },
+];
+
+export function encontraLegenda(label) {
```

Comment:
I think if we want to use this function as a Component we should use capital letter here and pass the label as prop


```js
export default function Legenda(label) {
 const legendaEncontrada = legendas.find(legenda => legenda.label === label);
 return legendaEncontrada && legendaEncontrada.legenda;
}
```

and use it like

```js
<Legend label={'something'} 
```

what do u think?

Reply:
How to use this component in the Graphics Bar component?
```js
<td><Legend label={'something'} /> </td>
```

Reply:
yeap, I think it's great :D